core/templates/dev/head/pages/exploration_editor/editor_tab/StateResponses.js

```diff
@@ -569,15 +569,15 @@ oppia.controller('StateResponses', [
           'stateInteractionIdService', 'stateCustomizationArgsService',
           'explorationContextService', 'editorContextService',
           'explorationStatesService', 'trainingDataService',
-          'AnswerClassificationService', 'focusService', 'DEFAULT_RULE_NAME',
-          'CLASSIFIER_RULESPEC_STR',
+          'textInputRulesService', 'AnswerClassificationService',
```

Comment:
Oh, thanks for the suggestion @seanlip. I'm not really familiar with Oppia's front-end (and, in fact Angular). Are you talking about something like a registry for rules service? Can you please point me to some code where we use such stuff? Thanks!